### PR TITLE
Clean up after pet

### PR DESCRIPTION
**Pull request by Bailey-Pearson**

### CODE REVIEW

**Review by jakeod99 — APPROVED** [review]
It seems as though the poop timer runs independently of when the poop gets cleaned, which is fine just not what I thought would be the case. It all looks good!

File: src/data/Pet.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+export const STATES = {
+    IDLE: 'idle',
+};
+
+export const THOUGHTS = {
+    HAPPY: 'happy',
+};
+
+export const COUNTDOWNS = {
+    POOP: 60,   //The pet poops every 60 seconds
```
Comment (jeremydelacruz) [inline src/data/Pet.js:15]: We love the poop constant

**Comment by jeremydelacruz** [comment]
Going off what Jake said, I guess we never really planned on how the subtraction timers would work, though I really have no preference whichever way we go. My initial thought was one global "tick" system where every stat would subtract at the same time for simplicity's sake. However, independent timers like the one you have here may mean more flexibility and that they don't depend on each other for anything. We can go off this example for the rest of the stats

File: src/data/Pet.js [hunk 2]
```diff
@@ -19,10 +23,42 @@ export class Pet {
...
+        this.poop = false;
     }
 
-    // ADD FUNCTIONS HERE TO AFFECT PET STATS
+    /**
+     * Called every second. Calls helper methods that update the pet after a certain number of seconds
+     * Returns true if the pet needs updated.
+     */
+    updatePet(){
+        if (this.poopCooldown()){   //Can just add methods to this if statement
```
Comment (jakeod99) [inline src/data/Pet.js:42]: I like this approach

**Review by jakeod99 — APPROVED** [review]
👍

File: src/data/Pet.js (hunk 2 above)
Comment (jakeod99) [reply src/data/Pet.js:42]: we could also just return the evaluation of the condition (like `return (this.poopCooldown())` but this may be easier to read

**Review by jeremydelacruz — APPROVED** [review]
I like it!

**Pull request merged**